tuf/api/metadata.py

```diff
@@ -891,18 +852,14 @@ class DelegatedRole(Role):
 
     Attributes:
         name: A string giving the name of the delegated role.
-        keyids: A set of strings each of which represents a given key.
-        threshold: An integer representing the required number of keys for that
-            particular role.
```

Comment:
these were removed because they are Role attributes, not DelegatedRole attributes (and we don't document parent attributes in other classes)